src/main/resources/templates/userinfo.mustache

```diff
@@ -145,6 +145,14 @@
                             {{social_security_record_claim_present}}
                         </dd>
                     </div>
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Return Code Claim Present
+                        </dt>
+                        <dd class="govuk-summary-list__value" id="user-info-return-code-claim">
+                            {{return_code_claim_present}}
```

Comment:
Might make testing easier if we were to display the entire claim, like we do for core identity?